ingest/a_vs_b_de.py

```diff
@@ -1,37 +1,36 @@
-#using https://nbviewer.org/github/theislab/diffxpy_tutorials/blob/master/diffxpy_tutorials/test/multiple_tests_per_gene.ipynb
+# using https://nbviewer.org/github/theislab/diffxpy_tutorials/blob/master/diffxpy_tutorials/test/multiple_tests_per_gene.ipynb
 import pandas as pd
 import scanpy as sc
 import diffxpy.api as de
 
 
-
 # TODO (SCP-5041): Extract to CLI arguments
-#Pairwise tests between groups
-#answers whether a given pair of groups shows differential expression for each gene
+# Pairwise tests between groups
+# answers whether a given pair of groups shows differential expression for each gene
 
-#arguments:
-#data: Anndata object, data matrix with cells x genes
+# arguments:
+# data: Anndata object, data matrix with cells x genes
 
-#grouping: str, column in data, the column that contains cell type labels. alternatively vector containing group labels
+# grouping: str, column in data, the column that contains cell type labels. alternatively vector containing group labels
 
-#test: type of test, default is z-test. other options: ’wald’, ’lrt’, ’t-test’, ’rank’
+# test: type of test, default is z-test. other options: ’wald’, ’lrt’, ’t-test’, ’rank’
 
-#lazy: bool, only possible if test is ztest, if true only evaluated once the user requests pval/coefficients for a specific pair of models
+# lazy: bool, only possible if test is ztest, if true only evaluated once the user requests pval/coefficients for a specific pair of models
 
-#noise_model: default is nb, specify NONE for wald and t test
+# noise_model: default is nb, specify NONE for wald and t test
 
 h5ad_file = "ingest/B_Plasma.h5ad"
 adata_file = sc.read_h5ad(h5ad_file)
 test = de.test.pairwise(
     data=adata_file,
     grouping="ClusterName",
-    #"rank" test correlates to Wilcoxon rank-sum
+    # "rank" test correlates to Wilcoxon rank-sum
     test="rank",
     lazy=False,
-    noise_model=None
- )
+    noise_model=None,
```

Comment:
Nice, didn't realize [python allowed trailing commas](https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list)